src/application/pokegear/configure/overlay_101_021EED44.c

```diff
@@ -0,0 +1,387 @@
+    FontID_Alloc(4, configureApp->heapId);
+    PokegearConfigure_LoadGraphics_Internal(configureApp);
+}
+
+static void PokegearConfigure_UnloadGraphics(PokegearConfigureAppData *configureApp) {
+    PokegearConfigure_UnloadGraphics_Internal(configureApp);
+    FontID_Release(4);
+}
+
+static void PokegearConfigure_CreateSpriteManager(PokegearConfigureAppData *configureApp) {
+    PokegearApp_CreateSpriteManager(configureApp->pokegear, GEAR_APP_CONFIGURE);
+    configureApp->contextMenuSpawner = TouchscreenListMenuSpawner_Create(configureApp->heapId, configureApp->pokegear->plttData);
+}
+
+static void PokegearConfigure_DestroySpriteManager(PokegearConfigureAppData *configureApp) {
+    TouchscreenListMenuSpawner_Destroy(configureApp->contextMenuSpawner);
+    PokegearApp_DestroySpriteManager(configureApp->pokegear);
+}
+
+static void PokegearConfigure_CreateSprites(PokegearConfigureAppData *configureApp) {
+    int i;
+
+    for (i = 0; i <= 4; ++i) {
+        configureApp->sprites[i] = SpriteSystem_CreateSpriteFromResourceHeader(configureApp->pokegear->spriteSystem, configureApp->pokegear->spriteManager, &sSpriteTemplates[i]);
+        thunk_Sprite_SetPriority(configureApp->sprites[i], 1);
+        Sprite_SetDrawFlag(configureApp->sprites[i], FALSE);
+        Sprite_SetAnimActiveFlag(configureApp->sprites[i], TRUE);
+    }
+    for (i = 5; i <= 8; ++i) {
+        configureApp->sprites[i] = SpriteSystem_CreateSpriteFromResourceHeader(configureApp->pokegear->spriteSystem, configureApp->pokegear->spriteManager, &sSpriteTemplates[i - 5]);
+        thunk_Sprite_SetPriority(configureApp->sprites[i], 1);
+        thunk_Sprite_SetDrawPriority(configureApp->sprites[i], 0);
+        Sprite_SetDrawFlag(configureApp->sprites[i], FALSE);
+        Sprite_SetAnimActiveFlag(configureApp->sprites[i], FALSE);
+    }
+}
+
+static void PokegearConfigure_DeleteSprites(PokegearConfigureAppData *configureApp) {
+    int i;
+
+    for (i = 0; i < 9; ++i) {
+        thunk_Sprite_Delete(configureApp->sprites[i]);
+    }
+}
+
+static void PokegearConfigure_CreateCursor(PokegearConfigureAppData *configureApp) {
+    PokegearCursorManager_AddButtons(configureApp->pokegear->cursorManager, sCursorButtons, NELEMS(sCursorButtons), 0, FALSE, configureApp->heapId, configureApp->sprites[0], configureApp->sprites[1], configureApp->sprites[2], configureApp->sprites[3]);
+    PokegearCursorManager_SetCursorSpritesDrawState(configureApp->pokegear->cursorManager, 1, FALSE);
+}
+
+static void PokegearConfigure_RemoveCursor(PokegearConfigureAppData *configureApp) {
+    PokegearCursorManager_RemoveCursor(configureApp->pokegear->cursorManager, 1);
+}
+
+static void PokegearConfigure_LoadContextMenuText(PokegearConfigureAppData *configureApp) {
+    MsgData *msgData;
+    int i;
+
+    configureApp->contextMenuItems = ListMenuItems_New(2, configureApp->heapId);
+    msgData = NewMsgDataFromNarc(MSGDATA_LOAD_DIRECT, NARC_msgdata_msg, NARC_msg_msg_0270_bin, configureApp->heapId);
```

Comment:
does this match if you merge the declaration with the assignment?

Reply:
i'm not entertaining that